pyxem/signals/pdf_profile.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+"""Signal class for Pair Distribution Function (PDF) radial profiles
+as a function of distance r.
+"""
+
+from hyperspy.signals import Signal1D
+import numpy as np
+
+
+class PDFProfile(Signal1D):
+    _signal_type = "pdf_profile"
+
+    def __init__(self, *args, **kwargs):
+        Signal1D.__init__(self, *args, **kwargs)
+
+    def normalise_signal(self):
+        """
```

Comment:
Need to say a little about how this is done here? Is it just dividing by the maximum of each PDF? Is that normal? Is there a reference for it?

Reply:
Thickness variation can be reduced by normalising the PDF to a common maximum. Will find a reference,